packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VideoController/VideoController.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, Button, Select } from 'grommet'
+import { CirclePlay, PauseFill } from 'grommet-icons'
+import FormattedTime from './FormattedTime'
+import Slider from './Slider'
+
+const VideoController = ({
+  isPlaying,
+  onPlayPause,
+  onSpeedChange,
+  played,
+  playbackRate,
+  duration,
+  onSliderMouseUp,
+  onSliderMouseDown,
+  onSliderChange
+}) => {
+  return (
+    <Box
+      background={{
+        dark: 'dark-3',
+        light: 'light-3'
+      }}
+    >
+      <Box pad={{ horizontal: 'xsmall', top: 'xsmall' }}>
+        <Slider
+          played={played}
+          onMouseUp={onSliderMouseUp}
+          onMouseDown={onSliderMouseDown}
+          onChange={onSliderChange}
+        />
+      </Box>
+
+      <Box direction='row' justify='between'>
+        <Box direction='row'>
+          <Box
+            a11yTitle='Video Play and Pause button'
+            alignSelf='center'
+            pad={{ horizontal: 'small' }}
+          >
+            <Button onClick={onPlayPause}>
+              {isPlaying ? <PauseFill /> : <CirclePlay />}
+            </Button>
+          </Box>
+
+          <Box a11yTitle='Video playback speed selection' width='140px'>
+            <Select
+              options={[0.25, 0.5, 1]}
+              value={playbackRate}
+              onChange={({ option }) => onSpeedChange(option)}
+              plain={true}
+            />
+          </Box>
+        </Box>
+
+        <Box
+          a11yTitle='Video player time played and total time'
```

Comment:
I'm not sure you need the label here, but maybe mark this up as a live region since the content is constantly updating. I'm not 100% sure about that, since it might be really annoying for a blind volunteer to here the time announced every second as it changes.

Grommet has an [AnnounceContext](https://v2.grommet.io/announcecontext) component for creating live regions, and setting the politeness level.

Reply:
The Grommet docs say that you can set the politeness level to 'polite', 'assertive' or 'off' but don't tell you what those different settings mean. Here's the [MDN docs for live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) (TLDR: use 'polite' unless it's really important for them to know the current video time every second.)